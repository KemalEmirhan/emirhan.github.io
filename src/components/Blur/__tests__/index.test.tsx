import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';
import Blur from '../';

describe('Blur', () => {
  it('should render', () => {
    const { container } = render(<Blur />);
    expect(container).toMatchSnapshot();
  });
});
