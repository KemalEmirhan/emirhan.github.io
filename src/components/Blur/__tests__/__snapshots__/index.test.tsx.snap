// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Blur > should render 1`] = `
<div>
  <div
    aria-hidden="true"
    class="top-blur pointer-events-none fixed left-0 top-0 z-9999 h-10 w-full cursor-none select-none opacity-95 backdrop-blur-[5px] inset-0 bg-[linear-gradient(to_bottom,rgba(252,252,252,1),rgba(252,252,252,0))] sm:h-20"
  />
</div>
`;
